tracer/src/Datadog.InstrumentedAssemblyGenerator/ProfilerMetadataImporter.cs

```diff
@@ -98,8 +100,12 @@ private void ImportInstrumentedAssemblyMetadata()
 
                     case MetadataTable.Field:
                     {
-                        var typeToAddField = OriginalModule.Types.SingleOrDefault(t =>
-                                                                                      t.Name.String.Equals(tokensAndName.Value.Type, StringComparison.InvariantCultureIgnoreCase));
+                        //TODO: should be SingleOrDefault but for now we don't have here the info about the parent type so we can find more than one type,
+                        // we choose the first one just to make it work.
+                        // https://datadoghq.atlassian.net/browse/DEBUG-1029
```

Comment:
Please remove JIRA link

Reply:
Fixed in 871a652d9b150d63443d87a542e34d4643145adf